src/sass/_instance_detail_page.scss

```diff
@@ -10,7 +10,8 @@
 
 .create-instance-from-snapshot-modal,
 .duplicate-instances-modal,
-.create-image-from-instance-modal {
+.create-image-from-instance-modal,
+.create-tls-identity {
```

Comment:
this feels a bit off adding identity related styling here, since the file is supposed to be dedicated for the instance detail page. Looking at the styles, it's a general modal dialog style override, I'd suggest move the styling into `styles.scss` and potentially come up with a more general class name. wdyt?

Reply:
Oh right, I see what you mean.
Yes, it is a general style. I have moved it into styles.scss for now, however do you think forms.scss would be appropriate? Not too sure, because although modals include forms, it is not directly relevant to forms themselves. Perhaps styles.scss is best as suggested.